code/modules/cargo/order.dm

```diff
@@ -1,3 +1,9 @@
+#define MANIFEST_ERROR_CHANCE 5
+// cargo error bitflags
+#define MANIFEST_ERROR_NAME (1 << 1)
+#define MANIFEST_ERROR_CONTENTS (1 << 2)
+#define MANIFEST_ERROR_ITEM (1 << 3)
```

Comment:
dmdoc this

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/63879/commits/b0a1ce1cdf6532c57c4de9450dfafa3c84f04fc7